test/lib/ufe/testAttribute.py

```diff
@@ -135,6 +176,35 @@ def runUndoRedoUsingMayaSetAttr(self, attr, newVal, decimalPlaces=None):
         cmds.redo()
         self.assertEqual(attr.get(), newVal)
 
+    def runMayaGetAttrTest(self, ufeAttr, decimalPlaces=None):
+        # Not all versions of Maya support getAttr with Ufe attributes.
+        if not self.getAttrSupportsUfe:
+            return
```

Comment:
Generic function used by different attribute type testing below. Will be silently skipped if getAttr does no support ufe.